openapi/components/headers/X-RateLimit-Remaining.yaml

```
@@ -0,0 +1,4 @@
+description: Remaining rate limit tokens for this request.
```

suggestion: to add more detail

I think it would be good to add `within the rate limit period`

```suggestion
description: Remaining number of rate limit tokens for this request within the rate limit period. 
```
and possibly an example 
```suggestion
description: |-
  Remaining number of rate limit tokens for this request within the rate limit period. 
  For example, in the sandbox environment, rate limits for non-GET endpoints are set at 3000 requests per 10 minutes.
```